server/src/main/java/info/doseamigos/authusers/DefaultAuthUserService.java

```diff
@@ -0,0 +1,79 @@
+package info.doseamigos.authusers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import info.doseamigos.amigousers.AmigoUser;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.HttpClients;
+
+import javax.annotation.Nonnull;
+import javax.inject.Inject;
+import java.io.IOException;
+import java.math.BigInteger;
+import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Implementation of AuthUserService using Google OAuth URLs.
+ */
+public class DefaultAuthUserService implements AuthUserService {
```

Comment:
The code works, but only locally.  With Amazon Lambda, it cannot connect to the DB.  I've had all sorts of headaches about this, and this isn't being used for the demo tomorrow.